build.sh

```diff
@@ -137,7 +137,7 @@ if buildAll || hasArg io || hasArg libengine || hasArg thirdparty; then
             cmake -DCMAKE_CXX11_ABI=ON ..
         else
             cd ${REPODIR}/thirdparty/cudf
-            git pull
+            # git pull
```

Comment:
so theres two sides to this sword. if you just comment it out. we make it less likely to detect when these changes WILL break things. While this is still an issue becuase we are using unexposed classes form CUDF we should have some kind of flag that we can set that decides whether or not it will update that. If it will update the cudf from source then you need to update the cudf in your conda as well to match. If you don't you have mismatch errors which are VERY confusing for new developers to blazing and has caused people to lose hours trying to build.